packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -975,7 +981,11 @@ mixin WidgetInspectorService {
   ///  * <https://github.com/dart-lang/sdk/blob/master/runtime/vm/service/service.md#rpcs-requests-and-responses>
   ///  * [BindingBase.initServiceExtensions], which explains when service
   ///    extensions can be used.
-  void initServiceExtensions(_RegisterServiceExtensionCallback registerServiceExtensionCallback) {
+  Future<void> initServiceExtensions(_RegisterServiceExtensionCallback registerServiceExtensionCallback) async {
```

Comment:
This doesn't have to be async and return a Future anymore then.

Reply:
done.